packages/core/errors/src/codes.ts

```diff
@@ -29,6 +29,10 @@
 export const WALLET_STANDARD_ERROR__REGISTRY__WALLET_NOT_FOUND = 3834000 as const;
 export const WALLET_STANDARD_ERROR__REGISTRY__WALLET_ACCOUNT_NOT_FOUND = 3834001 as const;
 
+// User-related errors.
+// Reserve error codes in the range [4001000-4001999].
+export const WALLET_STANDARD_ERROR__USER__REQUEST_REJECTED = 4001000 as const;
```

Comment:
@steveluscher this was originally 4001, inspired by [EIP-1193's errors](https://eips.ethereum.org/EIPS/eip-1193#provider-errors). Now it's completely arbitrary. Let me know if you want to change it.